submissions/akinbobolaFaith/dom/index.js

```diff
@@ -0,0 +1,63 @@
+let quantity = document.getElementById('quantity'),
+    price = document.getElementById('price'),
+    shipping = document.getElementById('shipping'),
+    cost = document.getElementById('cost'),
+    button = document.getElementById('myBtn'),
+    x = price.innerHTML,
```

Comment:
I think you should rename this variable. It is not self - explanatory @faithakinbobola